WalletWasabi.Gui/Controls/NoparaPasswordBox.cs

```diff
@@ -126,15 +125,15 @@ public NoparaPasswordBox()
 
 			try
 			{
-				var fontTester = SKTypeface.FromFamilyName(fontName);
-				if (fontTester.FamilyName == fontName)
-				{
+				
+				//if (fontTester.FamilyName == fontName)
+				//{
 					FontFamily = FontFamily.Parse(fontName); // Use the font.
-				}
-				else
-				{
-					throw new FormatException("font is missing fallback to default font");
-				}
+				//}
+				//else
+				//{
+				//	throw new FormatException("font is missing fallback to default font");
+				//}
```

Comment:
What's going on here? @molnard

Reply:
I see. That code made little. Anyhow removed the commented out code.

Reply:
Wait, FontFamily.Parse wouldn't throw in the same situation? Aah I don't understand this.

Reply:
"Return a new instance to a typeface that most closely matches the requested family name and style."
https://docs.microsoft.com/en-us/dotnet/api/skiasharp.sktypeface.fromfamilyname?view=skiasharp-1.68.0

Reply:
First I test if the font is in the system. I think Parse just silently fails but I don't remember exactly. Is it required to change the logic here?

Reply:
I made a small test: enter some fake name into Parse like SimSun121212. `Parse` ate it without exception and gives back the FontFamily: SimSun121212. Of course later this will cause problems. 

So the conclusion is: with `Parse` you cannot detect if the font is exist or not. With `SKTypeface.FromFamilyName` you can!

Reply:
Yes, the original code is important to have there, but it's deprecated, I think that's why @danwalmsley removed that. How can we do this in a clean way?

![image](https://user-images.githubusercontent.com/9156103/59092309-86c4bb00-8911-11e9-9584-db0364a2f110.png)